policies/trademark-policy.md

```diff
@@ -1,172 +1,325 @@
+You can use the Word Marks in book and article titles, and the Logo in illustrations within the document, as long as the use does not suggest that we have published, endorse, or agree with your work.
 
-- You may use the Word Marks in books and article titles, and the Logo in illustrations within a document, as long as the use does not suggest that we have published, endorse, or agree with your work.
+#### Events
 
-- Do not use CF Logos on the cover of a book or magazine without written permission.
+You can use the signage in the [Brand Standards][] to promote the Foundation and/or its Projects at events.
 
-- You may use CF Marks for private and personal use to make t-shirts, stickers, and caps for yourself and your friends (meaning people from whom you don’t receive anything of value in return).
+### Uses for which we are granting a license
 
-- You may use a CF Mark as a link to the home page of the applicable project or to a CF web site that is relevant to the reference.
+#### User groups
 
-### Visual Identity
+You can use the Word Marks as part of your user group name provided that:
 
-- Use only official versions of the Logo obtained from the Commonhaus Foundation or its projects.
+- The main focus of the group is the Foundation, its Projects, or its software;
+- Any software or services the group provides are without cost;
+- The group does not make a profit;
+- Any charge to attend meetings are to cover the cost of the venue, food and drink only.
 
-- You may not change any Logo except to scale it. This means you may not add decorative elements, change the colors, change the proportions, distort it, add elements, or combine it with other logos.
+Note that the general considerations for all uses, above, still apply.
+Specifically, you may not use or register the Marks as part of your own trademark, service mark, domain name, company name, trade name, product name or service name.
 
-- When the context requires the use of black-and-white graphics and the logo is color, you may reproduce the logo in a manner that produces a black-and-white image.
+#### Promotional goods
 
-## Commonhaus Trademarks and Crowdfunding
+"Promotional goods" are non-software goods that use the Marks and that are intended to advertise, promote, or show membership in the Foundation or its Projects.
 
-The Commonhaus Foundation recognizes that some projects may wish to use a project trademark when raising funds through crowdfunding platforms.
-These activities are not official foundation efforts, even if they are coordinated by core participants from our projects.
-Since crowdfunding operates externally to the Commonhaus Foundation, you must obtain permission before using a Commonhaus trademark on any crowdfunding page.
+You may make promotional goods for free giveaway at open source conferences and events using resources from our [Brand Standards][] site.
```

Comment:
```suggestion
You may make promotional goods for free giveaway at technology-related conferences and events using resources from our [Brand Standards][] site.
```

I don't think there's a good reason to limit this specifically to "open source conferences".